docs/CC2_PROTOCOL.md

```
@@ -309,31 +338,65 @@ After discovering the printer, establish an MQTT connection.
 
 ### Client ID Format
 
+**IMPORTANT**: Client ID must match the official web interface format.
+
 Generate a unique client ID:
 
 ```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)